.github/workflows/buildsCI.yaml

```diff
@@ -400,4 +400,46 @@ jobs:
        cmake --build . --config ${{env.LRS_BUILD_CONFIG}} -- -j4
        ls
   
-  
+  cppcheck:
+    name: cppcheck-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        
+      - name: Check_API
```

Comment:
No need for this stage here, we test it on Win & Linux, let's keep this static analysis only

Reply:
Removed